README.md

```diff
@@ -1,5 +1,7 @@
 # xVimscript
 
+[![Join the chat at https://gitter.im/exercism/xvimscript](https://badges.gitter.im/exercism/xvimscript.svg)](https://gitter.im/exercism/xvimscript?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
+
```

Comment:
Definitely... let's bring it in as "good enough" and immediately open a new PR.  Meh, just fix it and push it directly to master... I think this qualifies as "trivial".